ga4gh/backends.py

```diff
@@ -408,23 +420,27 @@ def searchVariantSets(self, request):
         """
         # This is temporary until we do this properly based on the output
         # buffer size
+        print(request)
         if request.pageSize is None:
             request.pageSize = 100
         response = protocol.GASearchVariantSetsResponse()
-        j = 0
+        currentIndex = 0
         if request.pageToken is not None:
-            j = int(request.pageToken)
-        v = []
-        while j < len(self._variantSetIds) and len(v) < request.pageSize:
-            vs = protocol.GAVariantSet()
-            vs.id = self._variantSetIds[j]
-            vs.datasetId = "NotImplemented"
-            vs.metadata = self._variantSets[vs.id].getMetadata()
-            v.append(vs)
-            j += 1
-        if j < len(self._variantSetIds):
-            response.nextPageToken = j
-        response.variantSets = v
+            currentIndex = int(request.pageToken)
+        variantSetList = []
+        print(self._variantSetIds)
+        while (currentIndex < len(self._variantSetIds) and
+               len(variantSetList) < request.pageSize):
+            variantSet = protocol.GAVariantSet()
+            variantSet.id = self._variantSetIds[currentIndex]
+            variantSet.datasetId = "NotImplemented"
+            variantSet.metadata = self._variantSetMap[variantSet.id]\
```

Comment:
pep8 passed all of the above. I have updated to hanging bracket or parens, and this also passes.